app/Application.php

```diff
@@ -151,11 +151,26 @@ protected function showHistory()
         do {
             $output = choice($this->getText('questions', 'export_question', ['export' => EXPORT_HISTORY, 'screen' => SCREEN]), [EXPORT_HISTORY, SCREEN]);
 
+            $exporter = 'historyConsoleExporter';
+
             if ($output === 'export') {
-                $nameOfFile = readline($this->messages['info']['name_of_file_create']);
+                $defaultFullPathName = 'export_' . date("m.d.y");
```

Comment:
you already gas logic to get the current date in the create new history after operation. Let's implement new helper function for it `now()`